loopy/check.py

```diff
@@ -952,44 +888,24 @@ def _check_for_unused_hw_axes_in_kernel_chunk(kernel, sched_index=None):
                     raise LoopyError("auto local tag encountered")
 
             if group_axes != group_axes_used:
-                if insn.id in boostable_insn_ids:
-                    warn("instruction '%s' does not use all group hw axes"
-                            " (available: %s used:%s). Loopy will generate code"
-                            " with the instruction executed along all the"
-                            " missing hw axes. This will result in an"
-                            " error from 2021.x onwards, calling"
-                            " loopy.add_inames_for_unused_hw_axes(...)"
-                            " might help in the transition."
-                            % (insn.id,
-                                ",".join(str(i) for i in group_axes),
-                                ",".join(str(i) for i in group_axes_used)),
-                            DeprecationWarning, stacklevel=2)
-                else:
-                    raise LoopyError("instruction '%s' does not use all group"
-                            " hw axes (available: %s used:%s)"
-                            % (insn.id,
-                                ",".join(str(i) for i in group_axes),
-                                ",".join(str(i) for i in group_axes_used)))
+                raise LoopyError(
+                        f"instruction '{insn.id}' does not use all group hw axes "
+                        "(available: %s used: %s). "
+                        "Calling loopy.add_inames_for_unused_hw_axes(...) "
+                        "might help."
+                        % (
+                            ",".join(str(i) for i in group_axes),
+                            ",".join(str(i) for i in group_axes_used)))
```

Comment:
Since we are breaking back compat. This necessitates a version bump.

Reply:
I assume you mean a package version bump. If so, done in ce3c4754.

Reply:
Yep, thanks!